### ISSUE TITLE
[Bug] Card not visible 

### ISSUE THREAD

**Issue by tjorim** [ngocjohn/vehicle-info-card#101]
### Short summary 📝

The card does not show with the latest b3 and b4 release 

### Detailed description 📋

I'm checking in the HA companion app on android and chrome on android. Haven't checked on pc to check the developer console. Beta 2 works, b3 and b4 do not (of version 1.6.1).

### Version 📦

v1.6.0

### Environment 🌐

Chrome on Android

**Comment by ngocjohn** [comment]
Hi @tjorim , I don't see any problem in the ios app with the latest beta b5, it is the final beta before the official release. Please test it, if there is any problem.. thanks

**Comment by tjorim** [comment]
It does not fix it, can reproduce b2 is okay, later versions are not.
I also tested in Chrome on Windows 11 which shows the same behaviour.
Console in DevTools shows this:
```
vehicle-info-card.js?hacstag=8105633501613:4324
Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'section_order')
    at yd._renderMainCard (vehicle-info-card.js?hacstag=8105633501613:4324:334)
    at yd.render (vehicle-info-card.js?hacstag=8105633501613:4320:66)
    at yd.update (vehicle-info-card.js?hacstag=8105633501613:22:238)
    at yd.performUpdate (vehicle-info-card.js?hacstag=8105633501613:11:4887)
    at yd.scheduleUpdate (vehicle-info-card.js?hacstag=8105633501613:11:4423)
    at yd._$ET (vehicle-info-card.js?hacstag=8105633501613:11:4331)
```

When setting up a new card it works (both in b2 and b5), the existing one does not.
Card was set up via UI, existing card does not have a extra_configs when checking the raw yaml.

**Comment by ngocjohn** [comment]
Thanks for the quick response.. i will try to fix this bug asap. 👍

**Comment by ngocjohn** [comment]
fix in 1.6.1 🎉
